### DISCUSSION TITLE
How do I get to a live stream with knowing the streamer but not following them?

### DISCUSSION THREAD

**Discussion by mqus** [twireapp/Twire#347] [Q&A] [answered]
I have tried searching for the streamer and I do get on their "profile page". This means I have access to the description, past broadcast and highlights tabs. But how do I open the livestream?

Is this just a missing feature or did I miss something?

**Comment by samfundev** [comment, marked as answer]
If a streamer is live, you can use the "Streams" tab of the search to find their live stream.
